modules/openapi-generator/src/main/resources/php-symfony/serialization/JmsSerializer.mustache

```diff
@@ -75,6 +75,12 @@ class JmsSerializer implements SerializerInterface
                     return $data + 0;
                 }
 
+                break;
+            case 'float':
+                if (is_float($data) || is_numeric($data)) {
+                    return (float) $data;
+                }
+
```

Comment:
thanks for the PR, which looks good.

i wonder if you can also add a case for `double` as well (above/before `float` case)